docs/element-templates/README.md

```diff
@@ -110,8 +112,10 @@ Templates are defined in template descriptor files as a JSON array:
 
 As seen in the code snippet a template consist of a number of important components:
 
+* `$schema : String`: Optional URI pointing towards the [json schema](https://json-schema.org/) which defines the structure of the element template json. Element template schemas are maintained in the [element templates json schema](https://github.com/camunda/element-templates-json-schema) repository. Following the [json schema](https://json-schema.org/) standard, you may use them to for validation or to get assistance (e.g., auto-completion) when working with them in your favorite IDE.
```

Comment:
> you may use them to for validation

one "to" too much

> you may use them for validation

Reply:
If we use "json" as a word, I'd go with

> JSON
> JSON schema

to make it consistent with the rest of the docs.

Reply:
:+1: ty